spinn_front_end_common/utility_models/live_packet_gather_machine_vertex.py

```diff
@@ -37,6 +37,8 @@ class LivePacketGatherMachineVertex(
                ('CONFIG', 1),
                ('PROVENANCE', 2)])
 
+    TRAFFIC_IDENTIFIER = "LPG_EVENT_STREAM"
```

Comment:
I don't quite see what this change has to do with this...

Reply:
just a quick harmless cleanup. Given future changes will use this.

Reply:
Hmm OK !